archive/har.d

```diff
@@ -1,3 +1,23 @@
+/**
```

Comment:
IMO there's almost never a case where /* should be used ; use /+  instead which allows nested comments

Reply:
I think I only see `/** ... */` in the compiler source...